fuzz/test-corpus.c

```diff
@@ -50,13 +50,13 @@ int main(int argc, char **argv) {
                 if (cpio_eof(cpio))
                     break;
 
-                printf("%s (%zu)\n", pathname, size);
+                printf("%s\n", pathname);
                 fflush(stdout);
                 OPENSSL_assert(pathname != NULL);
-                buf = malloc(size);
-                s = cpio_read(cpio, buf, size);
+                buf = malloc((size_t)size);
```

Comment:
Check for NULL here.
And why not use CRYPTO_malloc?

Reply:
The non-cpio code doesn't.  I guess @kroeckx wanted to minimize the exposure to libcrypto, to avoid possible other bugs to get in the way of checking the corpora.  On the other hand, the cpio reader uses OPENSSL routines, so that point is getting moot...